build.properties

```diff
@@ -1,7 +1,7 @@
 mc_version=1.20.1
 mod_version=3.2.0
 forge_version=47.1.65
-ccl_version=4.5.0.+
```

Comment:
CCL 1.20.1 is version 4.5.0

Reply:
The version on the maven is 4.4.0...  Is that changing?